python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -455,6 +455,12 @@ def transform(self, df):
 
                 elif (self.strategy == 'entity_embedding'):
 
+                    def get_embeddings(x, col):
+                        if int(self.__Enc[col][x]) < \
+                                np.shape(self.__weights[i])[0]:
+                            return self.__weights[i][int(self.__Enc[col][x]), :]
+                        return np.mean(self.__weights[i], axis=0)
+
```

Comment:
@NCherel  LGTY?

not sure about whether to use `self` or not.
`def get_embeddings(self, x, col):` The problem with doing so is that it may shadow the class `self` ...
Dunno 🤕